src/ResourceManager/Resources/Commands.Resources/help/Set-AzureRmPolicyDefinition.md

```diff
@@ -13,23 +13,31 @@ Modifies a policy definition.
 
 ## SYNTAX
 
-### SetByPolicyDefinitionName (Default)
+### NameParameterSet (Default)
 ```
-Set-AzureRmPolicyDefinition [-DisplayName <String>] [-Description <String>] [-Policy <String>]
+Set-AzureRmPolicyDefinition -Name <String> [-DisplayName <String>] [-Description <String>] [-Policy <String>]
```

Comment:
@mentat9 it looks like `-Name` is now mandatory in all parameter sets (except for the `-Id` set), which is a breaking change since there could be existing scripts that don't use the `-Name` parameter in their cmdlet call

Reply:
@mentat9 yep, sorry, forgot about this comment in the other PR 😃